portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
+function addRandomGreeting() 
+{
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Hello world!', 'This is my first Javascript function.', 'Surprise meme!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Increment greeting number
+  const greetingNum = Math.floor(Math.random() * greetings.length);
+  const greeting = greetings[greetingNum];
```

Comment:
consider removing this variable and replacing `greeting` with `greetings[greetingNum]` below.

Reply:
Done, I think it makes the code more compact/cleaner too. Although if `greeting` was used multiple times I would make keep it as a separate variable.